guides/v2.2/cloud/project/new-relic.md

```diff
@@ -137,27 +156,103 @@ Using the New Relic dashboard, you can immediately track and find:
 
-### New Relic and Blackfire
+### Configure notification channels
+
+To use the Adobe-generated alert policies to monitor your Production site, you must configure notification channels and map them to alert policies. Notifications about performance issues go out to all channels associated with an alert policy when conditions on the application or infrastructure trigger an alert. You also receive notifications when an issue is acknowledged and closed.
+
+New Relic provides templates for configuring different types of notification channels including email, Slack, PagerDuty, webhooks, and more. See the [Instructions for specific notification channels](https://docs.newrelic.com/docs/alerts/new-relic-alerts/managing-notification-channels/notification-channels-control-where-send-alerts#channel-types) in the New Relic documentation to review the prerequisites for using each type.
+
+The following instructions describe the high level steps to configure a Slack channel to receive New Relic alerts when the available disk space on your {{ site.data.var.ee }} Production server exceeds the low disk space threshold specified in the _Adobe Generated Disk Space Policy_ alert policy.
+
+{:.procedure}
+To configure a notification channel:
+
+1. Choose a notification channel type, and complete any prerequisite steps required to integrate the channel with the New Relic service.
+
+1. [Log in to your New Relic account](https://login.newrelic.com/login).
+
+1. Navigate to the New Relic APM page.
+
+1. On the New Relic APM Applications page, open the application for your Cloud environment.
+
+1. On the **Alerts** page, click **New notification channel**.
+
+   ![New Relic notification channel]({{ site.baseurl }}/common/images/cloud/cloud-new-relic-notification-channels.png){:width="600px"}
+
+1. Send a test notification to verify that the channel works.
+
+1. Map the notification to the Adobe-generated policies as needed.
+
+For detailed configuration instructions, see the following topics in the New Relic documentation:
+
+-  [Configure notification channels using the New Relic Alerts user interface](https://docs.newrelic.com/docs/alerts/new-relic-alerts/managing-notification-channels/notification-channels-control-where-send-alerts)
+
+-  [Configure notification channels using the New Relic API](https://docs.newrelic.com/docs/alerts/rest-api-alerts/new-relic-alerts-rest-api/rest-api-calls-new-relic-alerts#channels) Notification channels: Control where to send alerts)
+
+{:.bs-callout-warning}
+The Adobe-generated policies have default notification channels configured to notify Magento teams that support {{ site.data.var.ece }} customers.  Do not modify the configuration for these default channels, and do not remove them from the Adobe-generated alert policies.
+
+### Create Alert Policies
+
+Do not modify the Adobe-generated alert policies, as they might be automatically regenerated or improved over time.
```

Comment:
The be-verb in this sentence drives me crazy, but I'm not sure the best solution here. I will give it a go...

```suggestion
Do not modify the Adobe-generated alert policies, because they automatically regenerate or improve over time.
```